playbooks/10 - SP - Outbreak Response Framework/Investigate Outbreaks (Type All).json

```diff
@@ -0,0 +1,141 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "Investigate Outbreaks (Type All)",
+    "aliasName": null,
+    "tag": null,
+    "description": "Investigate all active Outbreak Alert on different SIEM tools, remediate and mitigate",
```

Comment:
Investigate all active Outbreak Alert --> alert is needed here?

Reply:
not required